src/js/components/Beacon/BeaconQueryUi.js

```diff
@@ -55,7 +60,7 @@ const BeaconQueryUi = () => {
 
   const assemblyIdOptions = beaconAssemblyIds.map((assembly) => (
     <Select.Option key={assembly} value={assembly}>
-      {assembly}
+      {t(assembly)}
```

Comment:
this will likely never have a translation

Reply:
I do not know, but doesn't hurt if any one would ever want to.